themes/default/content/blog/announcing-new-usage-based-pricing-for-your-whole-team/index.md

```diff
@@ -0,0 +1,38 @@
+
+Pricing based on the number of resources under management means that price scales naturally with the value of the cloud infrastructure complexity being tamed, managed and leveraged for your organization’s business goals via Pulumi.  Pricing is in terms of Pulumi Credits, and a Pulumi Credit is one [cloud resource]({{< relref "/docs/intro/concepts/resources" >}}) under management for one hour.
+
+New customers can sign up for Pulumi Team Edition starting today.  Existing customers can [contact us]({{< relref "/contact" >}}) now if they are interested in moving to the new Team or Enterprise Editions.
+
+The net result is three Pulumi editions available now:
+
+* **Individual Edition**: our Community Edition is being rebranded but remains unchanged, and is free for unlimited individual use.
+* **Team Edition**: up to 25 members can collaborate on projects with unlimited deployments, unlimited stacks and unlimited update history. This plan includes 150k free Pulumi Credits per month, letting you manage up to 200 cloud resources for free every month. After that, additional Pulumi Credits cost only $0.00025 each.
+* **Enterprise Edition**: for large organizations that need unlimited users, unlimited teams, Role-Based Access Control, SAML/SSO, policy enforcement, audit logging, and 12x5 or 24x7 support for their production deployments.
+
+Full details of the new editions are available on our [pricing page]({{< relref "/pricing" >}}).
+
+![Pricing Page](pricing-page.png)
+
+You can try out Pulumi with a 30-day free trial, no credit card required. If you need a longer trial period, want to discuss potential proof of concept projects, or are interested in advanced capabilities, such as policy and audit logs, Training, or Support, please [contact us]({{< relref "/contact" >}}).
+
+We remain committed to fostering a vibrant open source community while also delivering a robust production service that teams of all sizes are choosing for their cloud needs. From startups, to medium sized organizations, to the largest Fortune 500 customers, we truly believe that these new editions now offer a great option for everyone.
```

Comment:
```suggestion
We remain committed to fostering a vibrant open source community while also delivering a robust production service that teams of all sizes are choosing for their cloud needs. From startups, to medium sized organizations, to the largest Fortune 500 customers, we truly believe that these new editions now offer a great option for everyone. If you are using our pure open source offering, nothing has changed. We have also recently added a new offer for open source projects and foundations, and provide discounts for early stage startups. To discuss either, please contact us.
```

I don’t have the links handy for these — could someone add them?

Reply:
i added the open source free tier, and contact us.. wasn't sure what else you wanted to link to..